From b6fab5688dc41ce84bb65fa8d86fa075d314351b Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Mon, 23 Oct 2023 10:47:56 -0700
Subject: [PATCH 07/12] Do not disable unavailable plugin

Latest DNF complains about the `spacewalk` plugin being unavailable when
called by mock. Since it never was available in Clear, silence the
warning.

Rebased for mock 2.12 update by Patrick McCarty.
Rebased for mock 4.0 update by William Douglas.
Rebased for mock 5.2 update by William Douglas.

Signed-off-by: Patrick McCarty <patrick.mccarty@intel.com>
Signed-off-by: William Douglas <william.douglas@intel.com>
---
 py/mockbuild/config.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/py/mockbuild/config.py b/py/mockbuild/config.py
index c644594..eb2f483 100644
--- a/py/mockbuild/config.py
+++ b/py/mockbuild/config.py
@@ -301,7 +301,7 @@ def setup_default_config_opts():
     config_opts['system_dnf_command'] = '/usr/bin/dnf-3'
     config_opts['dnf_common_opts'] = ['--setopt=deltarpm=False', '--setopt=allow_vendor_change=yes', '--allowerasing']
     config_opts['dnf_install_command'] = 'install python3-dnf python3-dnf-plugins-core'
-    config_opts['dnf_disable_plugins'] = ['local', 'spacewalk', 'versionlock']
+    config_opts['dnf_disable_plugins'] = ['local', 'versionlock']
     config_opts["dnf_avoid_opts"] = {}
 
     config_opts['dnf5_command'] = '/usr/bin/dnf5'
-- 
2.42.0

